deck.json

```
@@ -1090,7 +1090,7 @@
             "__type__": "Note",
             "data": "",
             "fields": [
-                "Which gender (of the two \"main\" ones) seems to have a higher mortality rate",
+                "Which sex seems to have a higher mortality rate",
```

Thank you for making your first pull request 🎉
I do agree that the virus is unlikely to be much interested in a person's gender performance 😀 Nevertheless, hopefully avoiding a big debate, I would much prefer the previous formulation:
```suggestion
                "Which sex (of the two "main" ones) seems to have a higher mortality rate?",
```
Also, given that we're now using "sex", it also makes sense to say "Males" in the answer, no?

---

```suggestion
                "Which sex(biological) seems to have a higher mortality rate",
```

Gender appears to be the social role, while sex is biological. At that that's what wikipedia told me.

---

yes. There is a debate about how many genders there are, if gender is a social construct etc. However everyone agrees that there are two "main" sexes. The reason I say "main" is because sometimes due to defects people can have XXY or XYY chromosomes, which makes things a lot more confusing.

However, in a biological sense, XY means male sex, XX means female sex. There is not contraversy about biological sexes in terms of this. So saying "out of the two main ones is unnecessary"

---

The reason I find it important to maintain that mention of "main" sexes is the existence and struggles of [intersex individuals](https://www.wikiwand.com/en/Intersex).